src/codeu/chat/common/NetworkCode.java

```diff
@@ -31,6 +31,7 @@
       NEW_USER_RESPONSE = 12,
       NEW_CONVERSATION_REQUEST = 13,
       NEW_CONVERSATION_RESPONSE = 14,
+      REMOVE_CONVERSATION_REQUEST = 15,
```

Comment:
Since there's REMOVE_USER_RESPONSE and REMOVE_INTEREST_RESPONSE in [Server.java](https://github.com/lee-tammy/CodeU-Summer-2017/pull/11/files/7378526d6af24c4a287d81a413c8bcc0e31c71ab#diff-e2387988b5a78207609d160642f28cd3), I think it would make sense to add it.